notes/Module-2/2.4.4-Introduction_to_OAuth.md

```diff
@@ -0,0 +1,14 @@
+# Coach Notes
+
+## Coach comments
+Please add comments below which will help future coaches delivering this topic or suggestions for improvement of the content.
+
+|**Coach name**|**Delivery date**|**Comments**|
+|--------------|-----------------|------------|
+|Mandy|18/05/21|Learning objectives need to be modified|
+|Mandy|18/05/21|Assignment is not sufficient - to make the concept of OAuth more real, I got my students to sign up to the Spotify API and use their "Try in Console" feature to see how OAuth tokens are used to protect resources. It's great as you can see example cURL requests, simulate invalid tokens, expired tokens etc and see the Authorization Bearer tokens get issued and their expiry times. I would like to see this part of an assignment (as currently the assignment seems to just be copying the existing sequence diagram?) and get the students to create the UML diagrams based of their experience of using the Spotify API and as described in the [Spotify Implicit Grant Flow](https://developer.spotify.com/documentation/general/guides/authorization-guide/#implicit-grant-flow)|
```

Comment:
This seems like a great improvement. Can you add this to the session?

Reply:
done